.github/workflows/docker.yml

```
@@ -3,6 +3,8 @@ on:
   push:
     branches:
     - main
+    tags:
+    - '*'
```

Should we do the same as gems.yml and only run when creating a tag?
 
```suggestion
  create:
    tags:
      - v[0-9]+.[0-9]+.[0-9]+
```
```